.github/workflows/release-sass-api.yml

```diff
@@ -0,0 +1,33 @@
+# This has to be a separate workflow to satisfy pub.dev's somewhat draconian
+# requirements for when and how GitHub actions are allowed publish packages.
+# Specifically, it requires that each publish action be triggered by a tag that
+# contains the version number for that action, so we can't just publish sass_api
+# as part of the standard sass release flow because that was triggered by a tag
+# with the sass version number.
+name: Release sass-api
+
+on:
+  push:
+    tags: ['sass-api-[0-9]+.[0-9]+.*']
+
+jobs:
+  deploy_sass_api:
+    if: "github.event.repository.fork == false"
```

Comment:
Are these quotes necessary? IIRC quotes are only needed when the text contains a `*`

same below

Reply:
There are a number of active YAML characters. You're right that this doesn't need them, but I prefer to use them for anything with `${{ }}` interpolation just to avoid needing readers to remember when `{` is or isn't an active YAML character.